src/app/overview/overview.component.spec.ts

```diff
@@ -240,7 +246,7 @@ describe('OverviewComponent', () => {
     it('should get the select options', fakeAsync(() => {
       expect(component.filterData.length).toBeFalsy(0);
       component.beginPolling();
-      tick(1);
+      tick(tickTime);
```

Comment:
This tickTime confused me a little, but I've figured where it comes from. It's all good